src/scripts/main.js

```diff
@@ -22,7 +22,18 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const el in data) {
```

Comment:
better way for naming - make it explicit, if you already have variable "element", you can use word "item", for example. I approve your task, but in future, please, try to name your variables more explicit)